docs/manual/article-management/simple-tokens.de.md

```diff
@@ -0,0 +1,95 @@
+
+Beispiel:
+
+| Syntax              | Beschreibung                                              | Modul                       |
+| --------------------| --------------------------------------------------------- | --------------------------- |
+| `##tstamp##`        | Zeitstempel                                               |                             |
+| `##flang##`         | Aktuell verwendete Sprache                                |                             |
+| `##domain##`        | Aktuelle Domain                                           | Newsletter                  |
+| `##link##`          | Link zum Newsletter                                       | Newsletter                  |
+| `##channels##`      | Abonnierter News-Channel                                   | Newsletter                  |
+
+
+## Wo können Simple Tokens eingesetzt werden?
+- Benutzer: Modultyp 'Registrierung'
+- Benutzer: Modultyp 'Passwort vergessen'
+- Newsletter: 'Newsletterleser', 'Abonnieren' und 'Kündigen' (https://docs.contao.org/manual/de/core-erweiterung/newsletter/newsletter-verwaltung/#newsletter-personalisieren)
+- Extensions z. B. Notification-Center, Isotope, Catalog-Manager, Meta-Models, Contao-Leads
+- Als Platzhalter in Insert-Tags: https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes
+
+
+## Weitere Einsatzzwecke:
+`datei_von_##tstamp##.pdf` für Dateien erzeugt datei_von_1650437899.pdf
+
+`files/data/##form_broschuere##.pdf` Pfad zur PDF-Datei
+
+Es ist ebenso möglich, Simple Tokens als Fallabfrage zu verwenden. Hierzu werden Simple Tokens nicht mehr mit `##` geschrieben, sondern mit öffnender und schließender Klammer `{}`
+
+Beispiel für das Notification-Center:
+
+```
+Anfrage für: ##form_herkunft##
+{if form_termine!=""}
+   Termin: ##form_termine##
+{endif}
+
+{if form_formtyp!="erweitert"}
+  Name: ##form_name##
+  E-Mail: ##form_email##
+  Telefon: ##form_telefon##
+  {else}
+  Vorname: ##form_vorname##
+  Name: ##form_name##
+  Straße: ##form_strasse##
+  PLZ: ##form_plz##
+  Ort: ##form_ort##
+{endif}
+```
+
+Beispiel für das Newsletter-Modul:
+
+```
+{if flang==en}
+  Your language is English.
+{elseif==de}
+  Deine Sprache ist Deutsch.
+{else}
+  Couldn't assign a language.
+{endif}
+```
+
+
+## Welche Operatoren können bei Bedingungen gesetzt werden?
+
+| Syntax      | Beschreibung                  |
+| ----------- | ------------------------------|
+| `==`        | Expliziter Vergleich          |
+| `!=`        | Ungleich                      |
+| `===`       | Strikt mit Typ-Konvertierung  |
+| `<`         | Kleiner als                   |
+| `>`         | Größer als                    |
+| `<=`        | Kleiner gleich                |
+| `>=`        | Größer gleich                 |
+
+Generell empfiehlt es sich, keine Leerzeichen vor den Operatoren zu setzen. Mittels `||` und `&&` können auch mehrere Werte per AND/OR abgefragt werden:
```

Comment:
Ich nutze immer Leerzeichen, warum empfiehlt es sich nicht, die zu nutzen? Ist doch viel lesbarer :D Ich weiss nicht mehr genau aber kann sein, dass es in <4.12 nicht ging. Aber für alle die auf der aktuellen 4.13 sind, würde ich schon empfehlen, Leerzeichen zu nutzen.

Reply:
In Ergänzung zu Yanicks Anmerkungen: wenn es (in <4.12) nicht ging, wäre es m.E. korrekt zu sagen: "dürfen keine Leerzeichen verwendet werden". Die Formulierung "empfiehlt" impliziert für mich, daß beides geht, aber eines -- warum auch immer -- besser ist.

Reply:
Ich habe es jetzt einmal ausprobiert. Es funktioniert beides, sowohl mit Leerzeichen als auch ohne.

Reply:
@akroii funktioniert es auch in 4.9?

Reply:
@fritzmg ja, funktioniert auch in 4.9.

Reply:
Dann muss dieser Satz entfernt werden:

> Generell empfiehlt es sich, keine Leerzeichen vor den Operatoren zu setzen.

Reply:
erledigt